pkg/kubectl/cmd/util/helpers.go

```diff
@@ -64,6 +64,10 @@ func AddSourceToErr(verb string, source string, err error) error {
 
 var fatalErrHandler = fatal
 
+// ApplyAnnotation is the annotation used to store the previous configuration
+// of a resource that can be used in a three way diff by UpdateApplyAnnotation.
+const ApplyAnnotation = "k8s-kubectl-apply"
```

Comment:
Agreed per [this comment](https://github.com/kubernetes/kubernetes/pull/14621#discussion_r40510257), which identifies precedents.